Snowflake.Data/Core/FileTransfer/StorageClient/SFSnowflakeAzureClient.cs

```diff
@@ -268,7 +268,7 @@ private BlobClient GetUploadFileBlobClient(ref IDictionary<string, string>metada
 
             // Get the Azure client
             BlobContainerClient containerClient = blobServiceClient.GetBlobContainerClient(location.bucket);
-            return containerClient.GetBlobClient(location.key + fileMetadata.srcFileName);
+            return containerClient.GetBlobClient(location.key + fileMetadata.destFileName);
```

Comment:
RemoveFileName() instead ?

Reply:
I would leave it as is. It should be the same as RemoteFileName(), but this form would be more consistent with other cloud platform clients.